Tinylog/View Controllers/Tasks/TasksViewController.swift

```diff
@@ -708,65 +634,65 @@ final class TasksViewController: CoreDataTableViewController,
         editTaskViewController.delegate = self
         let nc: UINavigationController = UINavigationController(rootViewController: editTaskViewController)
         nc.modalPresentationStyle = UIModalPresentationStyle.fullScreen
-        self.navigationController?.present(nc, animated: true, completion: nil)
+        navigationController?.present(nc, animated: true, completion: nil)
     }
 
     func onClose(_ editTaskViewController: EditTaskViewController, indexPath: IndexPath) {
-        self.currentIndexPath = indexPath
-        self.tableView?.reloadData()
+        currentIndexPath = indexPath
+        tableView?.reloadData()
     }
 
     @objc func exportTasks(_ sender: UIButton) {
-        if self.list != nil {
-
-            do {
-
-                let fetchRequest: NSFetchRequest<NSFetchRequestResult> = NSFetchRequest(entityName: "Task")
-                let positionDescriptor  = NSSortDescriptor(key: "position", ascending: false)
-                let displayLongTextDescriptor  = NSSortDescriptor(key: "displayLongText", ascending: true)
-                fetchRequest.sortDescriptors = [positionDescriptor, displayLongTextDescriptor]
-                fetchRequest.predicate = NSPredicate(format: "list = %@", self.list!)
-                fetchRequest.fetchBatchSize = 20
-                let tasks: NSArray = try managedObjectContext.fetch(fetchRequest) as NSArray
-
-                var output: NSString = ""
-                var listTitle: NSString = ""
-                listTitle = self.list!.title! as NSString
-
-                output = output.appending(NSString(format: "%@\n", listTitle) as String) as NSString
-
-                for task in tasks {
-                    let taskItem: TLITask = task as! TLITask
-                    let displayLongText: NSString = NSString(format: "- %@\n", taskItem.displayLongText!)
-                    output = output.appending(displayLongText as String) as NSString
+        guard let list = list,
+            let title = list.title else {
+            return
+        }
+        
+        do {
+            let fetchRequest: NSFetchRequest<NSFetchRequestResult> = NSFetchRequest(entityName: "Task")
+            let positionDescriptor  = NSSortDescriptor(key: "position", ascending: false)
+            let displayLongTextDescriptor  = NSSortDescriptor(key: "displayLongText", ascending: true)
+            fetchRequest.sortDescriptors = [positionDescriptor, displayLongTextDescriptor]
+            fetchRequest.predicate = NSPredicate(format: "list = %@", list)
+            fetchRequest.fetchBatchSize = 20
+            let tasks: NSArray = try managedObjectContext.fetch(fetchRequest) as NSArray
+
+            var outputString: String = ""
+            outputString = outputString.appending("\(title)\n")
+            
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/sger/Tinylog/blob/61b2e19801c35844b857fed4761ebd1cb7f36195/Tinylog/View Controllers/Tasks/TasksViewController.swift#L662"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>TasksViewController.swift:662</code></td>
    </tr>
  </tbody>
</table>